src/ngraph_builder.cc

```diff
@@ -1724,12 +1724,29 @@ static Status TranslateFusedBatchNormOp(
     ng_y = make_shared<ng::op::GetOutputElement>(ng_batch_norm, 0);
     ng_mean = make_shared<ng::op::GetOutputElement>(ng_batch_norm, 1);
     ng_variance = make_shared<ng::op::GetOutputElement>(ng_batch_norm, 2);
+    // This is for Bessel's correction in ng_variance:
+    float ng_input_size = ng::shape_size(ng_input->get_shape());
+    float ng_scale_size = ng::shape_size(ng_scale->get_shape());
+    float sample_size = ng_input_size / ng_scale_size;
```

Comment:
`ng_input_size` and `ng_scale_size` are integer types. `sample_size` is also guaranteed to be integer type (since `ng_input_size%ng_scale_size == 0`
`sample_size_minus_one` can also be int.

I am think this: instead of introducing a vector division node in ngraph here: `auto Bess_scale = sample / minus_one;`
what is we do this:
```
float bess_scale = (float)(sample_size) / (float)(sample_size - 1);
std::vector<float> bess_scale_vector(ng::shape_size(ng_variance->get_shape()), sample_size);
auto ng_bess_scale = std::make_shared<ng::op::Constant>(ng_variance->get_element_type(), ng_variance->get_shape(), bess_scale_vector);
auto variance = ng_variance * ng_bess_scale;
```
(I suspect ngraph will have some kind of constant folding that would do the division at compile time, but am not sure. In that case the extra division node wouldn't matter much as it would be constant folded)


Other minor suggestions:
1. `ng_variance->get_shape()` and `ng_scale->get_shape()` are both `num_channels`. We can have a single variable and then use that to make it more readable.
2. We are using a `vector<float>` to initialize a `ng_variance->get_element_type()` type `ng::op::Constant`. Usually the type is expected to be `float` but could there be other types (at least other real types, if not integral types). Would they initialize well with `vector<float>` or would it complain?

Reply:
Thanks! changed